python/examples/minimal_build/Dockerfile.ubuntu

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-FROM ubuntu:focal
+FROM ubuntu:jammy
```

Comment:
We are already testing Ubuntu 20.04 on [test-ubuntu-20.04-python-3](https://github.com/ursacomputing/crossbow/actions/runs/9289313780/job/25562914452) .
I did update the minimal_build to 22.04 because there was a minor issue building and was easier to update than investigate to fix it and as we are already testing 20.04 I knew it was a CI set-up. I could investigate but I do think that we have to start updating our default jobs to 22.04 so I preferred to update it.